utils/saasherder.py

```diff
@@ -85,6 +86,22 @@ def _process_template(self, url, path, hash_length, target, parameters):
         resources = oc.process(template, target_parameters)
         return cluster, namespace, resources
 
+    def _collect_images(self, resources):
+        images = set()
+        for resource in resources:
+            try:
+                for c in resource["spec"]["template"]["spec"]["containers"]:
+                    images.add(c["image"])
+            except KeyError:
+                pass
+
+        return images
+
+    def _check_images(self, resources):
+        images = self._collect_images(resources)
+        for image in images:
+            bool(Image(image))
```

Comment:
This will have no effect... what do you want to do when an image does not exist?